modules/_labs/pagination/react/lib/Pages.tsx

```diff
@@ -0,0 +1,138 @@
+import styled from '@emotion/styled-base';
+import {IconButton} from '@workday/canvas-kit-react-button';
+import canvas, {borderRadius} from '@workday/canvas-kit-react-core';
+import _ from 'lodash';
+import React from 'react';
+
+const ActivePage = styled('button')({
+  display: 'inline-flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  fontSize: 13,
+  fontWeight: 700,
+  height: 32,
+  width: 32,
+  backgroundColor: canvas.colors.blueberry400,
+  color: canvas.colors.frenchVanilla100,
+  borderRadius: borderRadius.m,
+  ':focus': {outline: 'none'},
+});
+
+const NoPointerEvents = styled('div')({
+  pointerEvents: 'none',
+});
+
+const Ellipse = () => (
+  <NoPointerEvents>
+    <IconButton
+      key={'ellipse'}
+      aria-label={`Navigation Ellipse`}
+      variant={IconButton.Variant.Square}
+      size={IconButton.Size.Small}
+      tabIndex={-1}
+    >
+      ...
+    </IconButton>
+  </NoPointerEvents>
+);
+
+const Pages: React.FC<{
+  numPages: number;
+  currentPage: number;
+  clickHandler: (page: number) => void;
```

Comment:
Other components have an external API of `on*` vs `*Handler` Also props should be an interface.